base/BUILD.gn

```diff
@@ -3190,6 +3190,8 @@ target(gtest_target_type, "base_unittests") {
     data_deps += [
       ":base_unittests_bundle_data",
       "//cobalt/network:copy_ssl_certificates",
+    ]
+    deps += [
```

Comment:
@andrewsavage1 @oxve need some ideas here - how do i do this so that it can be configurable per platform ? The upstream code uses `third_party/icu:icudata` as a `deps` not `data_deps` - our code should conform.

We _may_ have some reasons to do it differently per platform, but i can't really think of a strong rationale.

Reply:
With how our bundling works we have to keep it in `data_deps` in order to be added to the content folder (and this bundles for the platforms that do bundling. It's not an issue to add it in `deps` but `data_deps` must be kept around.

Changing that will require somewhat significant refactoring.

This is true for all the places below where you've removed it from `data_deps` as well. I did a review of where it should be added when I made sure that only necessary dependencies are bundled.

Reply:
So the intent here is to eventually _not_ copy it to content folder at all. Perhaps the cleanest/safest way to get this landed is to just properly test on all platforms and make the switch all at once, not platform by platform. Then it just gets removed from `data_deps` entirely as i did here in this draft.

Reply:
that would be my preferred solution—in general, args.gn is a bad thing to rely on, and other solutions would mean modifying more upstream code than I'd be happy with

Reply:
Agreed. If that can be done easily it's the preferred solution.